hadoop-ozone/dev-support/checks/acceptance.sh

```diff
@@ -15,5 +15,6 @@
 # limitations under the License.
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 export HADOOP_VERSION=3
-"$DIR/../../../hadoop-ozone/dist/target/ozone-*-SNAPSHOT/compose/test-all.sh"
+OZONE_VERSION=$(cat $DIR/../../pom.xml  | grep "<ozone.version>" | sed 's/<[^>]*>//g'|  sed 's/^[ \t]*//')
```

Comment:
shellcheck:21: note: Double quote to prevent globbing and word splitting. [SC2086]
shellcheck:21: note: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead. [SC2002]